dashboard/src/pages/TreeDetails/TreeDetailsFilterList.tsx

```diff
@@ -22,38 +22,51 @@ const createFlatFilter = (filter: TFilter): string[] => {
 
 const TreeDetailsFilterList = ({
   filter,
-  onFilter,
 }: ITreeDetailsFilterList): JSX.Element => {
   const flatFilter = useMemo(() => createFlatFilter(filter), [filter]);
+  const navigate = useNavigate({ from: '/tree/$treeId' });
 
   const onClickItem = useCallback(
     (flatValue: string, _: number) => {
       const [field, value] = flatValue.split(':');
-      const newFilter = { ...filter };
-      newFilter[field as TFilterKeys][value] = false;
-      onFilter(newFilter);
+
+      const reallyNewFilter = JSON.parse(JSON.stringify(filter ?? {}));
```

Comment:
nit: we can use `newFilter` instead of `reallyNewFilter` here

Reply:
fixed in 
https://github.com/kernelci/dashboard/pull/93/commits/d1fd009e2c70dd340f5e557bc17ea6cf2b971c1d